core/src/components/avatar/test/shape/avatar.e2e.ts

```diff
@@ -95,5 +95,91 @@ configs({ directions: ['ltr'], modes: ['ionic-md'] }).forEach(({ config, screens
         await expect(avatar).toHaveScreenshot(screenshot(`avatar-shape-rectangular-image`));
       });
     });
+
+    test.describe('soft', () => {
+      test('should not have visual regressions when containing text', async ({ page }) => {
+        await page.setContent(
+          `
+          <style>
+            #container {
+              display: flex;
+              gap: 10px;
+            }
+          </style>
+
+          <div id="container">
+            <ion-avatar shape="soft" size="xsmall">AB</ion-avatar>
```

Comment:
I added the `xsmall` and `small` sizes to this shape's tests only because it has a different border radius than the other sizes.